tests/test_meshtastic_utils.py

```
@@ -3041,6 +3041,55 @@ def test_disconnect_ble_interface_none_input(self):
         # Should not raise any errors and return early
         _disconnect_ble_interface(None)
 
+    @patch("mmrelay.meshtastic_utils.logger")
+    def test_disconnect_ble_interface_client_none(self, mock_logger):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_logger`

(ARG002)